module/data/chat-message/request-message-data.mjs

```diff
@@ -36,7 +38,7 @@ export default class RequestMessageData extends ChatMessageDataModel {
       data: new ObjectField(),
       handler: new StringField({ required: true, blank: false }),
       targets: new ArrayField(new SchemaField({
-        actor: new ForeignDocumentField(foundry.documents.BaseActor),
+        actor: new DocumentUUIDField({ type: "Actor" }),
```

Comment:
I feel like we should add a validation here to prevent compendium UUIDs from being used. That would allow us to stick with `fromUuidSync` throughout the code while still allowing synthetic actors to be referenced. I can't think of a reason that compendium actors would be necessary since they generally can't roll anyway.